cartography/intel/aws/iam.py

```diff
@@ -80,7 +80,11 @@ def get_user_policy_data(boto3_session, user_list):
         name = user["UserName"]
         arn = user["Arn"]
         resource_user = resource_client.User(name)
-        policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        try:
+            policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        except resource_client.meta.client.exceptions.NoSuchEntityException:
+            logger.warning("Could not get user's policies: %s.policies.all() failed with NoSuchEntityException; skipping.", resource_user)
```

Comment:
The CI is complaining about the line being too long lol: `cartography/intel/aws/iam.py:86: [E501] line too long (138 > 120 characters)`.

Maybe do something like this:
```
logger.warning(
    f"Could not get policies for user {resource_user} due to NoSuchEntityException; skipping."
)
```

Btw to test the linter before pushing it up to the CI, you can do `pip install -r test-requirements.txt` and then `make test_lint`.

Let me know if you have trouble with the CLA, and should be good to go after that!

Reply:
Thanks for the tip. i'll fix up the linting issues - Had an extra issue with commas i didn't see. Will Push the update once linter has run successfully locally. 

Regarding CLA - I've already signed it but it still says it's failing. Not too sure what to do about this?